### ISSUE TITLE
Should Langevin integrators store work and heat in reduced units?

### ISSUE THREAD

**Issue by jchodera** [choderalab/openmmtools#163]
I think the `Langevin` integrators and nonequilibrium variants should track all energy quantities in dimensionless units (kT).

We did this in `perses` to simplify NCMC acceptance probabilities. I worry that it is far too easy to otherwise forget that the quantities we are dealing with are in the OpenMM-specific units of kJ/mol since the quantities can only be retrieved as unitless quantities.

Alternatively, or perhaps additionally, we should equip the integrators with retrieval methods that attach appropriate energy units, but I think it is just easier to always work in kT for work and heat quantities.

**Comment by jchodera** [comment]
Tagging @pgrinaway @gregoryross @bas-rustenburg to chime in here when able.

**Comment by bas-rustenburg** [comment]
I don't have a strong preference, but it could be useful to maintain compatibility with the OpenMM unit system. I think it's easier to forget that we are _not_ using OpenMM units.

**Comment by gregoryross** [comment]
I agree with @bas-rustenburg. While having energies in thermal units is theoretically appealing, I think this would be confusing for users familiar with other `openmm` integrators. Reporting the energies as ratios of kT means the values are dependent on temperature, which may lead to additional confusion.

**Comment by jchodera** [comment]
> I don't have a strong preference, but it could be useful to maintain compatibility with the OpenMM unit system.

The problem is that *all* OpenMM functions return unit-bearing quantities *except* context parameters.

If we add retrieval methods that return these quantities in appropriate units, I think that's fine. But expecting users to retrieve context parameters and remember what units they are in is just asking for trouble.

**Comment by pgrinaway** [comment]
I don't have a terribly strong preference either, but I think I do agree with @gregoryross that if this is to be used generally by OpenMM users, then just sticking with OpenMM units is probably best.

**Comment by bas-rustenburg** [comment]
@jchodera mentioned
>expecting users to retrieve context parameters and remember what units they are in is just asking for trouble. 

That is a solid point. It could get messy no matter what since it would require some form of memory of the units. Retrieval methods might be the way to go. 

We could even have two methods, one in kT, and one in OpenMM units, that way, it doesn't matter what we use internally.

One last point: 
If we do decide on an internal unit system, deviating from internal OpenMM units might require further computation just to convert units. It might be less optimal in terms of computation time, though I am not sure by how much.

**Comment by jchodera** [comment]
Let's go with adding getters that return unit-bearing quantities in the standard OpenMM unit system.

Who wants to tackle this?

**Comment by jchodera** [comment]
We still need someone to tackle the changes here to add a getter that attaches appropriate units. @pgrinaway, want to take this up since you were the first victim?
